packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -1,35 +1,31 @@
 import { css, cx } from '@emotion/css';
 import { isEmpty } from 'lodash';
-import React, { CSSProperties, ReactElement, ReactNode } from 'react';
+import React, { CSSProperties, ReactNode } from 'react';
 
-import { GrafanaTheme2, isIconName, LoadingState } from '@grafana/data';
+import { GrafanaTheme2, LinkModel, PanelModel, QueryResultMetaNotice, LoadingState } from '@grafana/data';
+import { selectors } from '@grafana/e2e-selectors';
 
 import { useStyles2, useTheme2 } from '../../themes';
-import { IconName } from '../../types/icon';
 import { Dropdown } from '../Dropdown/Dropdown';
 import { Icon } from '../Icon/Icon';
-import { IconButton, IconButtonVariant } from '../IconButton/IconButton';
+import { IconButton } from '../IconButton/IconButton';
 import { LoadingBar } from '../LoadingBar/LoadingBar';
-import { PopoverContent, Tooltip } from '../Tooltip';
+import { Tooltip } from '../Tooltip';
 
+import { PanelDescription } from './PanelDescription';
+import { PanelLinks } from './PanelLinks';
+import { PanelNotices } from './PanelNotices';
 import { PanelStatus } from './PanelStatus';
 
 interface Status {
   message?: string;
   onClick?: (e: React.SyntheticEvent) => void;
 }
 
-/**
- * @internal
- */
-export interface PanelChromeInfoState {
-  icon: IconName;
-  label?: string | ReactNode;
-  tooltip?: PopoverContent;
-  variant?: IconButtonVariant;
-  onClick?: () => void;
+interface Notices {
+  getPanelNotices: () => QueryResultMetaNotice[];
```

Comment:
Why the object and function here? Why not use the PanelStateInfo or TitleItem type , mainly wonder why we are passing an object with function that returns items instead of just passing the items.

Ideally we should pass a simple array of PanelChromeNotice type

Reply:
🤔 that's a good point, I remember following the pattern of passing a function down that was going to be called whenever we did hover on icons (i.e description, or panel links) to get variables interpolated, but double checking, in the case of panel notices this is not necessary. 

I will pass down the list of notices instead 😄